AirCasting/Graph/GraphView.swift

```diff
@@ -21,7 +21,7 @@ struct GraphView: View {
             
             ZStack(alignment: .topLeading) {
                 #warning("Replace dbStream with currently selected")
-                Graph(stream: session.dbStream!,
+                Graph(stream: session.measurementStreams?[0] as! MeasurementStreamEntity,
```

Comment:
I'm new, is it guaranteed at this point that `measurementStreams` is not empty?

Reply:
At this moment it should be guaranteed. We shouldn't be able to expand the session cart where the button triggering graph view is.
I'm attaching the design of the cart without measurements:
https://projects.invisionapp.com/d/main#/console/19745972/414476839/preview